ee/tabby-ui/package.json

```diff
@@ -11,7 +11,9 @@
     "preview": "next build && next start",
     "type-check": "tsc --noEmit",
     "format:write": "prettier --write \"{app,lib,components}/**/*.{ts,tsx,mdx}\" --cache",
-    "format:check": "prettier --check \"{app,lib,components}**/*.{ts,tsx,mdx}\" --cache"
+    "format:check": "prettier --check \"{app,lib,components}**/*.{ts,tsx,mdx}\" --cache",
+    "codegen": "graphql-codegen --config codegen.ts",
```

Comment:
can this step being merged into `next build` step? usually there'll be some webpack config for this

Reply:
maybe using a prebuild script is more convenient.